test/run_test_bort.sh

```diff
@@ -27,6 +27,108 @@
 ./test_bort_4 cmpmsg 1
 [ $? != 1 ] &&  exit 1
 
+# Check codflg().
+./test_bort_4 codflg 1
+[ $? != 1 ] &&  exit 1
+
```

Comment:
Shouldn't this script be in the test_scripts subdirectory along with all of the other test scripts, rather than in the test directory with all of the source code?

Reply:
Well that is open to discussion.

All the scripts in test_scripts are for utils, and I was planning on moving all of them to a test_utils directory.

Reply:
Actually, the test_outtest.sh script (wrapper script for all of the outtest*F90 codes) is also in test_scripts, so it's not just scripts for testing utilities in that directory.

That said, if you wanted to have a separate test_utils directory for all of the utility scripts, and then a separate test_scripts directory containing just your new test_bort.sh script and the test_outtest.sh script, I wouldn't be opposed to that ;-)